apps/schools/api_views/school.py

```diff
@@ -44,7 +44,7 @@ class SchoolsList(KLPListAPIView, CacheMixin):
     search_fields = ('name', 'id', 'dise_info__dise_code',)
 
     def get_queryset(self):
-        qset = School.objects.filter(status=2).distinct('id')
+        qset = School.objects.filter(id__gt=63929, status=2).distinct('id')
         get_geom = self.request.GET.get('geometry', 'no')
```

Comment:
Hardcoded School ID for Odisha schools. Again, should be replaced by an State filter as mentioned above.